release-notes-6.0.0.md

```diff
@@ -0,0 +1,11 @@
+#### Version Number
+${version-number}
+
+#### Breaking Changes
+- D1021267: The buildenv-dotnet-mono-image has been deprecated and removed.
```

Comment:
Somewhat surprisingly I don't actually consider the discontinuation a project to be a breaking change, but it's obviously debatable so if you prefer to mark this up as a break then I'm fine with it.